vm/src/obj/objsequence.rs

```diff
@@ -127,6 +128,10 @@ impl<T: Clone> PySliceableSequence for Vec<T> {
     fn len(&self) -> usize {
         self.len()
     }
+
+    fn is_empty(&self) -> bool {
```

Comment:
This probably is an infinite recursion?

Reply:
Given how `len` is implemented I doubt it, but, just in case, perhaps some turbofish could be used here to call the real method?